src/couch_stats/src/couch_stats_sup.erl

```diff
@@ -29,6 +29,8 @@ init([]) ->
         {
             {one_for_one, 5, 10}, [
                 ?CHILD(couch_stats_server, worker),
+                ?CHILD(csrt_server, worker),
+                ?CHILD(csrt_logger, worker),
```

Comment:
Since CSRT lives in couch_stats it should be prefixed with `couch_stats_`. I know don't always follow that rule, but most applications use the `$app_...` prefix scheme.